samples/sensor/mhz19b/boards/nucleo_g0b1re.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2021, G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&usart1 {
```

Comment:
you should probably introduce a shield or describe how is the sensor connected. this board doesn't seem to have this sensor.